src/ui/RunEditor.tsx

```diff
@@ -1729,6 +1721,29 @@ export class RunEditor extends React.Component<Props, State> {
         this.update(tab);
     }
 
+    private shouldShowTab(tab: Tab) {
+        if (tab === Tab.RealTime || tab === Tab.GameTime) {
+            return true;
+        }
+
+        const gameInfo = getGameInfo(this.state.editor.game);
+        if (gameInfo == null) {
+            return false;
+        }
+        if (tab === Tab.Variables || tab === Tab.Rules) {
```

Comment:
We may want to show the variables despite there not being any game info (yet). Because otherwise you can't use the custom variables.